sonar-helm-for-iac/src/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/Masterminds/sprig/v3 v3.2.3
 	github.com/gobwas/glob v0.2.3
 	github.com/samber/mo v1.11.0
-	github.com/sonarsource/go/src v1.21.8-4
+	github.com/sonarsource/go/src v1.21.8-7
```

Comment:
I wonder why `go.work.sum` is not updated and still contains `github.com/sonarsource/go/src v1.21.8-3 h1.....` 🤔

Reply:
We probably don't even need it anymore, since we went with keeping forked dependency in a separate repo. I'll see (as a separate ticket) if it can/should be removed or how to maintain it properly if it's needed.